template/build/webpack.test.conf.js

```diff
@@ -1,12 +1,12 @@
 'use strict'
 // This is the webpack config used for unit tests.
 
-const utils = require('./utils')
-const webpack = require('webpack')
-const merge = require('webpack-merge')
-const baseWebpackConfig = require('./webpack.base.conf')
+var utils = require('./utils')
```

Comment:
is there any reason to use `var`, not `const`?

Reply:
I though it was used before my changes, but it looks like its switched to use strict and const. I'll change it back 👍